.circleci/src/jobs/@embed-jobs.yml

```diff
@@ -0,0 +1,75 @@
+embed-init:
+  working_directory: ~/audius-client
+  docker:
+    - image: circleci/node:14.18
+  steps:
+    - checkout
+
+    - create_concatenated_package_lock:
+        filename: combined-package-lock.txt
+
+    - create_concatenated_patch_file:
+        filename: combined-patch-file.txt
+
+    - restore_cache:
+        keys:
+          - cache-{{ checksum "package-lock.json" }}-{{ checksum "combined-package-lock.txt" }}-{{ checksum "combined-patch-file.txt" }}
+
+    - restore_cache:
+        keys:
+          - cache-{{ .Revision }}
+
+    - run:
+        name: Link packages
+        command: |
+          npm run link
+
+    - persist_to_workspace:
+        root: ./
+        paths:
+          - node_modules
+          - packages/embed/node_modules
+          - packages/stems/node_modules
+          - packages/stems/dist
+
+
+embed-build-staging:
+  working_directory: ~/audius-client
+  docker:
+    - image: circleci/node:14.18
```

Comment:
@sliptype Is this something for us to keep an eye on with the node upgrade?

Reply:
Yup, this is find for now. Will do a follow up pr to bring embed up to 18!